mkdocs.yml

```diff
@@ -39,6 +39,7 @@ pages:
         - Choosing a build image: ci/build-image.md
         - Specifying branches/tags to build: ci/specify-branches.md
         - Working with env variables: ci/env-vars.md
+        - Setting git options: ci/set-git-options.md
```

Comment:
Let's title this "Using shallow clone and git options". I think the page title should say "shallow clone" somewhere so it's searchable.